frontend/packages/insights-plugin/src/components/AdvisorChart/index.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { ChartDonut, ChartLabel, ChartLegend } from '@patternfly/react-charts';
+import { global_Color_dark_200 as globalColorDark200 } from '@patternfly/react-tokens';
+
+import {
+  AdvisorChartTitle,
+  AdvisorChartLegendIcon,
+  chartColorScale,
+  riskLabels,
+} from './chartHelpers';
+
+const AdvisorChart = ({ metrics, clusterId }) => {
+  const { t } = useTranslation();
+
+  const issues = Object.entries(metrics);
+  const issuesCount = Object.values(metrics).reduce<number>(
+    (acc: number, cur: number) => acc + cur,
+    0,
+  );
+
+  return (
+    <ChartDonut
+      data={issues.map(([k, v]) => ({
+        label: `${v} ${t(riskLabels[k])}`,
+        x: k,
+        y: v,
+      }))}
+      title={`${issuesCount}`}
+      subTitle={
+        issuesCount === 1
+          ? t('insights-plugin~Total issue', { count: issuesCount })
+          : t('insights-plugin~Total issues', { count: issuesCount })
```

Comment:
Hey - There's no need to add a check here; if you just use the `count` keyword in `t` and run `yarn i18n` it will update the strings correctly.

Reply:
@rebeccaalpert, wow, that works like a charm. Added one more commit :)